tests/integration/api_v2/subscribeApi/transactionEvent.test.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
```

Comment:
`tests/integration/api_v2/subscribeApi/transactionEvent.test.js` to `tests/integration/api_v2/events/transactions.test.js`